ruby/conditional_modifiers.md

```diff
@@ -0,0 +1,87 @@
+# Limit use of conditional modifiers to short, simple cases
+
+Conditional modifiers (i.e., `if` or `unless` at the end of a line) be
+surprising when they appear on long or complex lines. The reader might not see
+them while scanning the code.
+
+So, prefer to use them only for short, simple cases. For example:
+
+```ruby
+do_later if async?
+```
+
+The example above can read more naturally than:
+
+```rb
+if async?
+  do_later
+end
+```
+
+## Complex conditions
+
+However, if the line is too long (around 80 characters) or complex (e.g., an
+`if` with multiple conditions `if a && b`) prefer the multi-line form:
+
+```ruby
+# Avoid
+do_something if condition && condition2
+
+# Prefer
+if condition && condition2
+  do_something
+end
```

Comment:
I'd love a "real world"-like example here. "condition && condition2" might be too abstract for an example

Reply:
I added a new one. LMK what you think